appveyor.yml

```diff
@@ -29,6 +29,9 @@ environment:
 
   - R_VERSION: devel
 
+before_build:
+  - cmd: ECHO no Revision metadata added to DESCRIPTION
+  #translate from unix: - cmd: ECHO "Revision:" $CI_BUILD_REF >> ./DESCRIPTION
```

Comment:
some win user please push to this branch corresponding windows command so packages in appveyor will also contain Revision field. You also need to change env var name to appveyor commit env var.

Reply:
Do we have any Windows developers on the project?   Or do you mean a Windows user.  I'm not really following.   Does this need to happen before merging,  or can I just merge the PR now.

Reply:
@mattdowle We can merge the first commit from this branch, other ones are failed attempts to append line to DESCRIPTION file on windows. Appveyor does not build drat repo so it is not as much important for this PR.